Source/Plugins/Core/com.equella.core/js/__mocks__/SearchResult.mock.ts

```diff
@@ -16,15 +16,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import * as OEQ from "@openequella/rest-api-client";
+import * as _ from "lodash";
```

Comment:
is all of lodash needed in this scope?
Would
```suggestion
import { range } from "lodash";
```

work? (would also need a change from `_.range` to `range` below, on line 312)

Reply:
I'd say so. I just replaced like for like. I'll give it a tweak. :wink: